util.py

```diff
@@ -189,16 +189,13 @@ def extract_tagged_values(data, name, key_split_char='.', tag_list=None):
         return [(dict(), value) for value in values.values()]
 
     if type(values) is not dict:
-        return _extract_tagged_value(name, values, key_split_char, tag_list)
+        return (_extract_tags(name, key_split_char, tag_list), values)
```

Comment:
this should probably be `[(_extract_tags(name, key_split_char, tag_list), values)]` to keep the return type (list) consistent.  should have been like that before too